### PR TITLE
OZ-49: Using bleeding edge Common Reports and Ozone Config.

### PR DESCRIPTION
**Pull request by Ruhanga**
This PR is updating artefacts making up moving parts to their SNAPSHOT versions (_Common Reports_ and _Ozone Configuration_).
Ticket: https://mekomsolutions.atlassian.net/browse/OZ-49

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
   <properties>
     <!-- Dependency versions -->
-    <commonreportsVersion>1.2.0</commonreportsVersion>
+    <commonreportsVersion>1.3.0-SNAPSHOT</commonreportsVersion>
```
Comment (mks-d) [inline pom.xml:22]: Why that one specifically?

**Review by mks-d — COMMENTED** [review]
This should probably be associated with a ticket connected to the next version, no?

File: pom.xml [hunk 2]
```diff
@@ -19,9 +19,9 @@
...
     <odoo_initializerVersion>2.1.0</odoo_initializerVersion>
```
Comment (mks-d) [inline pom.xml:23]: What about this one then?

Comment (Ruhanga) [reply pom.xml:23]: I could change that one as well. I wasn't quite aware how much of a moving part it is.

Comment (Ruhanga) [reply pom.xml:23]: Not much has changed  on `odoo_initializer` though.

**Review by mks-d — APPROVED** [review]

**Pull request merged**